Dockerfile

```diff
@@ -27,7 +29,8 @@ COPY --chown=carma . /home/carma/src/
 RUN ~/src/docker/checkout.bash -b ${GIT_BRANCH}
 RUN ~/src/docker/install.sh
 
-FROM setup
+# Final image
+FROM setup AS final
```

Comment:
Is this a change to match best practices for Dockerfiles?

Reply:
Yeah, it's a change to match best practice. Docker builder also complained about it when building